app/src/main/res/values/arrays.xml

```diff
@@ -39,6 +39,24 @@
         <item>always</item>
     </array>
 
+    <array name="message_expiring_descriptions">
+        <item>@string/nc_expire_message_off</item>
+        <item>@string/nc_expire_message_four_weeks</item>
+        <item>@string/nc_expire_message_one_week</item>
+        <item>@string/nc_expire_message_one_day</item>
+        <item>@string/nc_expire_message_eight_hours</item>
+        <item>@string/nc_expire_message_one_hour</item>
+    </array>
+
+    <array name="message_expiring_values">
+        <item>expire_0</item>
+        <item>expire_2419200</item>
+        <item>expire_604800</item>
+        <item>expire_86400</item>
+        <item>expire_28800</item>
+        <item>expire_3600</item>
+    </array>
```

Comment:
see above I would expect you only need the values in here.

Reply:
yep